apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/SwapWebView.tsx

```diff
@@ -112,7 +112,7 @@ const SwapWebView = ({
   const locale = useSelector(languageSelector);
   const redirectToHistory = useRedirectToSwapHistory();
   const enablePlatformDevTools = useSelector(enablePlatformDevToolsSelector);
-
+  const demo1 = useFeature("ptxSwapLiveAppDemoOne");
```

Comment:
Actually this should be changed to using the manifest id after the internal demo?